src/main/java/appeng/core/sync/packets/PacketPaintedEntity.java

```diff
@@ -20,50 +20,57 @@
 
 
 import io.netty.buffer.ByteBuf;
-import io.netty.buffer.Unpooled;
 
-import net.minecraft.entity.player.EntityPlayer;
+import cpw.mods.fml.common.network.simpleimpl.MessageContext;
 
 import appeng.api.util.AEColor;
 import appeng.core.sync.AppEngPacket;
-import appeng.core.sync.network.INetworkInfo;
+import appeng.core.sync.AppEngPacketHandler;
 import appeng.hooks.TickHandler;
 import appeng.hooks.TickHandler.PlayerColor;
 
 
-public class PacketPaintedEntity extends AppEngPacket
+public class PacketPaintedEntity implements AppEngPacket, AppEngPacketHandler<PacketPaintedEntity, AppEngPacket>
 {
 
-	private final AEColor myColor;
-	private final int entityId;
+	private AEColor myColor;
+	private int entityId;
 	private int ticks;
 
 	// automatic.
-	public PacketPaintedEntity( ByteBuf stream )
+	public PacketPaintedEntity()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)